plugins/network-elements/nuage-vsp/src/com/cloud/network/element/NuageVspElement.java

```diff
@@ -454,6 +563,80 @@ protected boolean applyACLRules(Network network, List<? extends InternalIdentity
     }
 
     @Override
+    public boolean applyNetworkACLs(Network config, List<? extends NetworkACLItem> rules) throws ResourceUnavailableException {
```

Comment:
Since this is an overridden method from a base class, we can not change the return type